src/quo/components/inputs/amount_input/style.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo.components.inputs.amount-input.style
+  (:require
+    [quo.foundations.colors :as colors]
+    [react-native.platform :as platform]))
+
+(defn get-selection-color
+  [customization-color theme]
+  (colors/alpha
```

Comment:
this should instead be -
```
(defn get-selection-color
  [customization-color theme]   
(colors/resolve-color customization-color
                         theme   
   (if platform/ios? 1 0.2)))
```
as resolve color handles the opacity with the 3rd parameter. The 50/60 values are also handled internally within the resolve color function so there is no need to do that there.

Reply:
Done ✅